src/reference/asciidoc/kafka.adoc

```diff
@@ -2153,6 +2164,35 @@ class Listener implements ConsumerSeekAware {
 ----
 ====
 
+To make things simpler, version 2.3 added the `AbstractConsumerSeekAware` class, which keeps track of which callback is to be used for a topic/partition.
+The following example shows how to seek to the last record processed, in each partition, each time the container goes idle.
+
+====
+[source, java]
+----
+public class SeekToLastOnIdleListener extends AbstractConsumerSeekAware {
+
+    @KafkaListener(id = "seekOnIdle", topics = "seekOnIdle")
+    public void listen(String in) {
+        ...
+    }
+
+    @Override
+    public void onIdleContainer(Map<org.apache.kafka.common.TopicPartition, Long> assignments,
+            ConsumerSeekCallback callback) {
```

Comment:
Is this argument out of use now?
What is the point to perform a search if we have this argument here?
Or what is the relationship between this argument and whatever is `assignments`?

Thanks for understanding.

Reply:
Doh; of course; the collection is really only needed when performing seeks from some other arbitrary environment (like in the example above this).

Will polish.